proxy/http/HttpConfig.h

```diff
@@ -670,6 +671,7 @@ struct OverridableHttpConfigParams {
   MgmtInt default_buffer_size_index;
   MgmtInt default_buffer_water_mark;
   MgmtInt slow_log_threshold;
+  MgmtInt ssl_client_verify_server;
```

Comment:
We have a use case for permissive mode for which ssl_client_verify_server has a third value.